xdebug_stack.c

```diff
@@ -619,6 +619,11 @@ void xdebug_init_debugger(TSRMLS_D)
 			XDEBUG_LOG_PRINT(XG(remote_log_file), "I: Checking header 'REMOTE_ADDR'.\n");
 			XDEBUG_ZEND_HASH_STR_FIND(PG(http_globals)[TRACK_VARS_SERVER], "REMOTE_ADDR", HASH_KEY_SIZEOF("REMOTE_ADDR"), remote_addr);
 		}
+
+		if (remote_addr && strstr(XDEBUG_ZEND_HASH_RETURN_VALUE(remote_addr), "://")) {
+			XDEBUG_LOG_PRINT(XG(remote_log_file), "W: Invalid remote address provided containing URI spec '%s'", XDEBUG_ZEND_HASH_RETURN_VALUE(remote_addr));
+			remote_addr = NULL;
+		}
```

Comment:
Please add a new line here.

Also I wonder whether we can test for this case, by using the ``HTTP_REMOTE_ADDR`` env var (or something similar) to simulate this happening?